test/clearingHouse/ClearingHouse.accounting.xyk.test.ts

```diff
@@ -571,19 +568,21 @@ describe("ClearingHouse accounting verification in xyk pool", () => {
         it("funding-induced liquidation", async () => {
             // maker add liquidity
             await addOrder(fixture, maker, 100, 10000, lowerTick, upperTick)
-
             // taker open
             await q2bExactInput(fixture, taker, 150)
 
+            await accountBalance.mockMarkPrice(baseToken.address, parseEther("4"))
             // set index price to let taker pay funding fee
             mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
                 return [0, parseUnits("4", 6), 0, 0, 0]
             })
 
-            // taker pays funding
-            while ((await clearingHouse.getAccountValue(taker.address)).gt(0)) {
-                await forwardBothTimestamps(clearingHouse, 3000)
+            // taker is not bankruptcy yet, even he has loss
+            expect(await clearingHouse.getAccountValue(taker.address)).to.be.gt(0)
```

Comment:
account value > 0 might still be liquidatable. probabbly needs to use `_isLiquidatable` 's logic to make sure that this account right now is healthy.

Reply:
Fixed in 2fa7e1f6904d49042e96ef6d89c54adc43c15334